extlinks/links/admin.py

```diff
@@ -1,10 +1,21 @@
 from django.contrib import admin
+from django.contrib.contenttypes.admin import GenericTabularInline, GenericStackedInline
 
 from .models import URLPattern, LinkSearchTotal, LinkEvent
 
 
+class LinkEventURLPatternAdminInline(GenericTabularInline):
+    model = LinkEvent
+    show_change_link = True
+    exclude = ['username', 'user_id', 'on_user_list', 'url']
```

Comment:
Can we make the rest of the link event fields read-only? Right now, my UI loads most but not all of the link events because there are a lot of them.